dask_jobqueue/core.py

```diff
@@ -195,6 +198,8 @@ def __init__(self,
             extra = dask.config.get('jobqueue.%s.extra' % self.scheduler_name)
         if env_extra is None:
             env_extra = dask.config.get('jobqueue.%s.env-extra' % self.scheduler_name)
+        if python is None:
+            python = sys.executable
```

Comment:
I think that we can safely include this directly in the signature.  It's not going to change.

Reply:
So this means no config entry?

Reply:
Right.  If we were to include this as part of the config (which I don't think is necessary) then we would want to check the default every time we called the constructor here, so we would need this check to be within the constuctor as you have it here.  However if the default value is just going to be `sys.executable` (which I think is best today) then we can include it as a default directly in the signature of the constructor.  This is a bit better both because it reduces code, and because it makes the default visible if people look at the docstring.